lab-jessica/lib/challenge.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
I see it's working as intended, and solving for the least number of moves! There's something that I'm having a hard time putting my finger on though, and I feel like I'm missing something that may be an issue. 

That being said, well done!